image/docker_schema1.go

```diff
@@ -112,6 +113,17 @@ func (m *manifestSchema1) ConfigBlob() ([]byte, error) {
 	return nil, nil
 }
 
+// Config returns the image configuration as per OCI v1 image-spec
+func (m *manifestSchema1) Config() (*imgspecv1.Image, error) {
+	// Best effort to have an OCI configuration from a docker v2s1 manifest.
+	// we don't compute diffIDs or other v2s2 things here.
```

Comment:
I'll document this in the API in types.go, I don't really want to go thru schema1-schema2-oci conversions. People using this API won't actually look for those layers info, rather stuff like Entrypoint and such